e2e-test/reconcile_namespace_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eo pipefail
+
+test_dir=e2e-test
+
+source ${test_dir}/functions.sh
+
+e2e-test-setup
+
+WATCH_NAMESPACE=espejo start-espejo
+
+sleep 5s
```

Comment:
What is this `sleep` waiting for?

Reply:
To wait until the operator has started up (in the background) and done some initial reconciles.
Do you have a better idea?